src/backend/src/services/tasks.services.ts

```diff
@@ -180,17 +195,26 @@ export default class TasksService {
       };
     });
 
-    const updatedTask = await prisma.task.update({
-      where: { taskId },
-      data: {
-        assignees: {
-          set: transformedAssigneeUsers
-        }
-      },
-      ...taskQueryArgs
+    const updatedTask = taskTransformer(
+      await prisma.task.update({
+        where: { taskId },
+        data: {
+          assignees: {
+            set: transformedAssigneeUsers
+          }
+        },
+        ...taskQueryArgs
+      })
+    );
+
+    const assigneeSettings = await prisma.user_Settings.findMany({ where: { userId: { in: newAssignees } } });
+    assigneeSettings.forEach(async (settings) => {
+      if (settings.slackId) {
+        await sendSlackTaskAssignedNotification(settings.slackId, updatedTask);
+      }
```

Comment:
I think you can make this a helper method in utils that takes in a list of userIds and a task (since you do it here and in lines 87-91)